module1.js

```diff
@@ -14,12 +14,14 @@
 */
 
 function question1(name1) {
-  if (name1) {
-    // TODO:
-    return;
-  }
+  return name1 === "Code Your Future" ? true : false;
```

Comment:
Close :) `name1 === "Code Your Future"` already gives you a boolean, so the rest of the line is redundant